src/esp/bindings_js/index.js

```diff
@@ -1,12 +1,26 @@
-/*global Module */
+/* global FS, Module */
 
 import WebDemo from "./modules/web_demo";
 import VRDemo from "./modules/vr_demo";
 import "./bindings.css";
 
-Module["onRuntimeInitialized"] = function() {
-  console.log("hsim_bindings initialized");
+Module.preRun.push(() => {
+  let arg;
+  let scene;
+  for (arg of window.location.search.substr(1).split("&")) {
+    let [key, value] = arg.split("=");
+    if (key == "scene") {
+      scene = value;
+    }
+  }
+  FS.createPreloadedFile("/", scene, scene, true, false);
```

Comment:
Does this cache it once it is loaded?

Reply:
In chrome inspector, I see that it is served from the disk cache.